tests/sentry/eventstream/kafka/test_consumer.py

```diff
@@ -649,13 +648,14 @@ def _get_producer(self, topic):
     def setUp(self):
         super().setUp()
         self.events_topic = f"events-{uuid.uuid4().hex}"
+        # self.transactions_topic = f"transactions-{uuid.uuid4().hex}"
         self.commit_log_topic = f"events-commit-{uuid.uuid4().hex}"
         self.override_settings_cm = override_settings(
+            KAFKA_EVENTS=self.events_topic,
+            KAFKA_TRANSACTIONS=self.events_topic,
```

Comment:
This was the line that I put back - running the combined version of the post process forwarder (entity="all") requires that events and transactions share a Kafka topic

Reply:
Thanks!